.claude/skills/create-issue/SKILL.md

```
@@ -0,0 +1,21 @@
+---
+name: create-issue
+description: Draft a GitHub Issue as the spec artifact for Spec-Driven Development (SDD) before implementing a feature or bug fix in this project.
+---
+
+This project uses Spec-Driven Development (SDD): discuss in Plan mode first, create a GitHub Issue as the spec artifact, then implement. Always offer to draft an issue before writing code.
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)